Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Clothing.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.3</ArmorRating_Sharp>
```

Comment:
For cloth:
```
<StuffPower_Armor_Sharp>0.01</StuffPower_Armor_Sharp>
<StuffPower_Armor_Blunt>0.015</StuffPower_Armor_Blunt>
```
With CE, it's essentially `<ArmorRating_Sharp> = Apparel Thickness * <StuffPower_Armor_Sharp>` so Sharp armor should probably be set to around 0.05.

Reply:
Fixed in 2666b70